Formula/qcli.rb

```diff
@@ -17,7 +17,7 @@ class Qcli < Formula
   depends_on "pkg-config" => :build
   depends_on "ffmpeg"
   depends_on "qt@5"
-  depends_on "qwt"
+  depends_on "qwt-qt5"
```

Comment:
Can we switch this to `qt` and keep `qwt` instead?

Reply:
I don't know!
Should I ?